packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,205 @@
+//
+// SIDE-NAV
+//
+
+@use "../mixins/focus-ring" as *;
+
+// WIP, naming, etc. will be refined... TODO: Add JSON for color tokens, etc.
+:root {
+  /* Sizes: */
+  --token-side-nav-item-padding-horizontal: 12px;
+
+  /* Colors (WIP): */
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+
+  // Non-active "Links"
+  --token-side-nav-surface-default: #0c0c0e;
+  --token-side-nav-surface-hover: #3b3d45;
+  --token-side-nav-surface-active: #656a76;
+
+  // "Active" "Links"
+  --token-side-nav-selected-surface-default: #656a76;
+  --token-side-nav-selected-surface-hover: #3b3d45;
+  --token-side-nav-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  height: 100vh;
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default);
+}
+
+// Child "wrapper" blocks & child components:
+
+// * Sub-component: Header
+// ---------------------------------------
+.hds-side-nav__header {
+  display: flex;
+  gap: 8px; // TODO: replace with margins
+  align-items: center;
+  padding: 12px;
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
```

Comment:
I started adding these background colors to the various components so they could be displayed more easily as stand alone components in the showcase. I'm not sure if that's a good idea to repeat on all the child elements though vs. relying on the background color of the sidebar container itself.

Reply:
the `__header` should be responsible only of the layout

Reply:
re. the showcase, if we need a background for contrast we have to add it via custom CSS in the  CSS for the showcase page